src/routes/docs/advanced/self-hosting/+page.markdoc

```diff
@@ -8,7 +8,7 @@ Appwrite was designed from the ground up with self-hosting in mind. You can inst
 
 # System requirements {% #system-requirements %}
 
-Appwrite is designed to run well on both small and large deployments. The minimum requirements to run Appwrite are as little as **1 CPU core** and **2GB of RAM**, and an operating system that supports Docker.
+Appwrite is designed to run well on both small and large deployments. The minimum requirements to run Appwrite are as little as **1 CPU core** and **2GB of RAM**, with **2GB of swap memory**, and an operating system that supports Docker.
```

Comment:
I'm not sure we should say Appwrite **requires** 2 GB of swap memory. The bare minimum to run Appwrite is 2GB of memory. However, you may be doing stuff that requires more memory so you would either increase memory or add swap. It's on the developer to decide which they want.